src/Microsoft.AspNetCore.Mvc.Core/ModelBinding/CompositeModelBinder.cs

```diff
@@ -36,70 +36,77 @@ public CompositeModelBinder(IList<IModelBinder> modelBinders)
         /// <inheritdoc />
         public IList<IModelBinder> ModelBinders { get; }
 
-        public virtual Task<ModelBindingResult> BindModelAsync(ModelBindingContext bindingContext)
+        public virtual Task BindModelAsync(ModelBindingContext bindingContext)
         {
             if (bindingContext == null)
             {
                 throw new ArgumentNullException(nameof(bindingContext));
             }
 
-            var newBindingContext = CreateNewBindingContext(bindingContext);
-            if (newBindingContext == null)
-            {
-                // Unable to find a value provider for this binding source. Binding will fail.
-                return ModelBindingResult.NoResultAsync;
-            }
-
-            return RunModelBinders(newBindingContext);
+            return RunModelBinders(bindingContext);
```

Comment:
@rynowak do you believe this was covered with your measurements, or do you think this happens often enough to be measured and changed specifically?